vulnfeeds/cves/versions_test.go

```diff
@@ -1078,9 +1085,14 @@ func TestValidateAndCanonicalizeLink(t *testing.T) {
 			},
 			wantCanonicalLink: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=ee1fee900537b5d9560e9f937402de5ddc8412f3",
 			wantErr:           false,
+			skipOnCloudBuild:  true, // observing indications of IP denylisting as at 2025-02-13
 		},
 	}
 	for _, tt := range tests {
+		if _, ok := os.LookupEnv("BUILD_ID"); ok && tt.skipOnCloudBuild {
+			t.Logf("test %q: skipping on Cloud Build", tt.name)
+			continue
+		}
 		t.Run(tt.name, func(t *testing.T) {
```

Comment:
This logic could probably be moved inside of `t.Run` and use [`t.Skipf`](https://pkg.go.dev/testing#T.SkipNow)

Reply:
Thanks for the tip! Adjusted in https://github.com/google/osv.dev/pull/3154/commits/938da4e7c0265bea343ddb75065bf4619dc647e1